gradle/versions.gradle

```diff
@@ -23,16 +22,16 @@ ext.plugs = [
 // Dependencies
 ext.deps = [
         // compile
-        "appcompatv7" : "com.android.support:appcompat-v7:$supportLibraryVersion",
         "kotlinStdlib": "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion",
-        "gson"        : "com.google.code.gson:gson:2.6.2",
-
-        // androidTestCompile
-        "runner"      : "com.android.support.test:runner:0.4.1",
-        "rules"       : "com.android.support.test:rules:0.4.1",
+        "gson"        : "com.google.code.gson:gson:2.8.1",
 
         // testCompile
         "junit"       : "junit:junit:4.12",
         "spockCore"   : "org.spockframework:spock-core:1.1-groovy-2.4",
+
+        // integration - if you change these, you need to update the values in ./.buildscript/test.sh
```

Comment:
@benjamin-bader Added this comment to help future developers to ensure these dependencies remain untouched. If they are touched, the developer needs to update the current dex count values in the `test.sh`.